### ISSUE TITLE
Balkan Federation

### ISSUE THREAD

**Issue by settintotrieste** [settintotrieste/Victoria-2-Cold-War-Enhancement-Mod-CWE#55]
Dragonbreath6434 suggested:

Hello again, I was wondering if at any point you planned on adding more to the Balkan Federation events, say if all goes well and Bulgaria joins, perhaps deals could be made with the USSR, Albania, Romania,and Greece to become the larger Balkan Federation that was envisioned, but slowed due to the Tito-Stalin split. Just an idea, I say this as I had to do some of my own edits when I eventually united the Balkans under Yugoslavia, making some requirments and such, to do some type of equivalent to the Balkan Fed. None the less, I'm sure there's a lot of other things you want to get done for the mod :P Upload.wikimedia.org

**Comment by ARR8** [comment]
Something I would like to add here is that Yugo+Bulgaria is already lacking in flavor even without adding any other states. For example, when they are combined, Bulgarian becomes the biggest culture, yet does not separate when the rest of the Yugoslav states do, making it easy for a player to keep Yugoslavia alive with Bulgaria. I'm sure the Bulgarians would have resented being subservient to an area much poorer than they are, see Catalonia, Flanders, etc.
  I can envision the Bolgars taking leadership post-Tito and the Serbs rebelling, much like the Croats, Bosniaks, etc. did against the Serbs in our time. I'm sure an Orthodox Christian vs. Orthodox Christian conflict would also have de-emphasized the religious conflicts that led to such as violent breakup of Yugoslavia in our time.

**Comment by ARR8** [comment]
With regards to commit dbac63b, I'm m not sure the Balkan Federation should have formation mechanics like the other unions, especially since it's nearly impossible to get even a greater Yugoslavia to become a great power. Instead, it probably makes more sense with the situation of the time for an event chain following a successful Yugo+Bulgaria merger, like the first user suggested, where the Soviet Union demands ideological and other concessions from Yugoslavia in exchange for allowing it greater integration with the Soviet sphere.

There should probably be several events with difficult choices and loss of autonomy and independence in exchange for Albania, etc. and then similar difficult diplomacy with America and the NATO Balkan states to consider integration. The USSR would not want to give up puppet states to an independent state, so there would need to be concessions and possibly creation of a puppet in the future for the USSR to grant these lands to Yugoslavia. The challenge would probably be to keep neutrality even as the Soviet Union puts pressure on Yugoslavia to fully join the bloc and Sovietize, but doing so would prevent expansion to any of the other Balkan states.

Furthermore, even the masterfully-formed Balkan Federation should have a lot of trouble staying together post-Cold War, even more difficult than keeping just Yugoslavia together. And there should of course be flavor and ethnic issues along the way, like I suggested in the earlier comment. One possible example in addition to those could be an event where Albania refuses to join despite the USSR's consent, which leads to a small diplomatic crisis.

I'm not sure the current system would help player immersion in the mod. It seems to me that a greater Balkan federation should be a hard-earned reward with lots of difficulties along the way, and the eventual purpose of a Yugoslavia game, sort of like a Danubian Federation game in many of the other Vic2 mods. The current system would require gamey tactics or prestige spam to become a great power and then taking one decision to unite the Balkans which remain stable forever.

This would, of course, be a lot of events, but IMO Yugoslavia is in the perfect historical position to create a very interesting diplomatic game where you have to balance pressure from superpowers and from within, and that's definitely something I'd like to see happen.

**Comment by settintotrieste** [comment]
That is a good idea and I think it can complement the current set up well. Unfortuantely, I won't be able to touch on this anytime soon as I will need to start clearing up the US-Soviet Joint projects and a few other complex features for 1.06. Though if anyone wishes to contribute to this, it will be greatly appreciated!

**Comment by settintotrieste** [comment]
Further feedback:

Playing as Yugoslavia, I get the opportunity to invite Bulgaria to become a federal republic of Yugoslavia, but then nothing ever happens. I've tried going through that event several times with restarts and everything, but I don't ever get any kind of answer or acknowledgement that I've invited them. Is it even possible for Bulgaria to become a part of Yugoslavia, or do you have to go through the Balkan Federation route?

I feel that the Balkan Federation is an unrealistic and uninteresting project and would much rather just absorb the last missing part of the union of South Slavs and be satisfied with it.

**Comment by ARR8** [comment]
Events 8216020-8216025 in [united_balkans.txt](/settintotrieste/Victoria-2-Cold-War-Enhancement-Mod-CWE/blob/3c6eb3c2a6e01241fd250b7129170508de3ac261/CWE/events/united_balkans.txt#L162) all concern Bulgaria. I recall at some point there was a 10% chance for the Soviets to accept the unification, but it seems that was removed.
  Somewhat confusingly, the Soviets only get the event for unification a month after it is proposed, and no one is notified upon rejection. A simple fix for this specific issue would be to put a chance for acceptance back and add an event notifying the Yugoslavs upon rejection. That can be a patch for this specific problem until there's time/information for a more realistic rewrite.
  However, it does seem that, between when and I played and when they disabled the union, the NWO folks fixed the problems with Buglaria never getting independence from Serbia, so that can be considered a non-issue.

Additionally, it seems the Albanian events in that same file never get activated by the AI, and the events leading to them are counterintuitive. That doesn't need to be immediately fixed, though.

For what it's worth, I agree that much more work needs to be put into Yugoslavia before the Balkan Federation should be considered. Most of the Yugoslav events concern its downfall, and there are only a few about its interesting diplomatic position.
